api/src/main/resources/workbench.yaml

```diff
@@ -780,47 +767,24 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
 
-  /v1/admin/users/id-verification/list:
+  /v1/admin/users/list:
     get:
       tags:
         - profile
       description: >
           Returns a list of profiles for users to be reviewed. Requires
           REVIEW_ID_VERIFICATION authority.
```

Comment:
Did we change the authority name here?

Reply:
no - was wondering if i should do that in this PR or a follow up PR